packages/kbn-optimizer/limits.yml

```diff
@@ -124,3 +124,4 @@ pageLoadAssetSize:
   sessionView: 77750
   cloudSecurityPosture: 19109
   visTypeGauge: 24113
+  unifiedSearch: 49195
```

Comment:
Can we decrease the limit of the data plugin somewhere below 100KB?

Reply:
yep, updated